### ISSUE TITLE
Maven failures

### ISSUE THREAD

**Issue by TheBusyBiscuit** [TheBusyBiscuit/builds#2]
>[WARNING] Could not transfer metadata com.sk89q.worldedit:worldedit-bukkit:6.1.1-SNAPSHOT/maven-metadata.xml from/to paper-repo (https://repo.destroystokyo.com/repository/maven-public/): sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

This is suddenly happening all over the place.
I will have to investigate this further at some point, no projects compile due to this.

**Comment by Redemption198** [comment]
It seems to be related with a missing certificate, I've searched on Google and I found this:

> Go to URL in your firefox browser, click on HTTPS certificate chain (next to URL address). Click  `"more info" - "security" - "show certificate" - "details" - "export.."`. Pickup the name and choose file type example.cer. Now you have file with keystore and you have to add it to your JVM
Determine location of cacerts files, eg.  `C:\Program Files (x86)\Java\jre1.6.0_22\lib\security\cacerts`.
Next import the example.cer file into cacerts in command line:
`keytool -import -alias example -keystore  C:\Program Files (x86)\Java\jre1.6.0_22\lib\security\cacerts -file example.cer`
You will be asked for password which default is `changeit`
Restart your JVM/PC.
source: http://magicmonster.com/kb/prg/java/ssl/pkix_path_building_failed.html

[Source](https://stackoverflow.com/questions/21076179/pkix-path-building-failed-and-unable-to-find-valid-certification-path-to-requ)

Hope it helps.

**Comment by TheBusyBiscuit** [comment]
I know and I found the same, also various other results that show the same.
But I'm not on Windows, this runs on a raspberry pi (Raspbian), the Java installation on my Windows pc works just fine.
But on Linux it happened very suddenly, 

Also, it's not with this particular URL, it happens with every URL, no matter what.
And I cannot really go about any add every URL to my certificates...
Also checked the cacerts and it's integrity looks fine. I already tried various proposed solutions but either the Java version for Raspbian didn't include the tool they suggested or it did but just didn't work at all.

**Comment by Redemption198** [comment]
The keytool should be inside the bin folder (PathToThisFolder\jdk1.8.0_171\bin\) and it should be available on every Java installation.

**Comment by Redemption198** [comment]
As written [here](https://letsencrypt.org/docs/certificate-compatibility/), the Let's Encrypt certificate (the one used by repo.destroystokyo.com) is supported by Java 8 >= 8u101, maybe you have to update your JDK.

**Comment by TheBusyBiscuit** [comment]
I was already running the latest version. And reinstalling did not help either.

But in the end I got it working now.
I updated my linux kernel and switched from oracle jdk to openjdk which at least gave me access to the keytool with which I was able to import the certificates.

This has definitely been a huge hassle...
And cost me weeks of waiting and hours of updating and googling.

But it is working again, so thank you nonetheless.
